WcaOnRails/config/locales/en.yml

```diff
@@ -505,6 +506,7 @@ en:
         qualification_results_reason: "Please elaborate here why you would like to use qualification times. Please fill this out in English!"
         event_restrictions: ""
         event_restrictions_reason: "Please elaborate here why you would like to restrict certain event combinations. Please fill this out in English!"
+        main_event: "The winner of the competition will be the winner of this event."
```

Comment:
nit: I think flipping this sentence around actually makes more sense from a cause-effect perspective:

"The winner of this event will be the winner of the competition"

Reply:
Sounds good, changing!